Jens/update vignette (#313)
NorskRegnesentral/shapr: vignettes/understanding_shapr.Rmd

# Plot the resulting explanations for observations 1 and 6
plot(explanation, plot_phi0 = FALSE, index_x_explain = c(1, 6))
```

> There are multiple plot options speficified by the `plot_type`-argument in `plot`. The `waterfall` option will show the changes in the prediction score due to each features contribution (their Shapley values): 

Reviewer: ```suggestion
There are multiple plot options speficified by the `plot_type` argument in `plot`. The `waterfall` option shows the changes in the prediction score due to each features contribution (their Shapley values): 
```

```{r}
plot(explanation, plot_type = "waterfall", index_x_explain = c(1, 6))
```

> This final two plot options, `"beeswarm"` and `"scatter"` can be useful when you have many observations that you want to explain. For the purpose of illustration we will explain the whole `airquality` dataset (including the training data) for these plot types. 

Reviewer: ```suggestion
The final two plot options, `"beeswarm"` and `"scatter"`, can be useful when you have many observations that you want to explain. For the purpose of illustration, we explain the whole `airquality` dataset (including the training data) for these plot types. 
```

> The `plot_type = "beeswarm"` summarises the distribution of the Shapley values along the x-axis for all the features. Each point gives the shapley value of a given instance, where the points are colored by the feature value of that instance:

Reviewer: ```suggestion
The `plot_type = "beeswarm"` summarises the distribution of the Shapley values along the x-axis across all features. Each point gives the Shapley value of a given instance, where the points are colored by the feature value of that instance:
```

```{r}
x_explain <- data[, ..x_var]
explanation <- explain(
  model = model,
  x_explain = x_explain,
  x_train = x_train,
  approach = "empirical",
  prediction_zero = p0
)
plot(explanation, plot_type = "beeswarm")
```

> Lastly, the `plot_type = "scatter"` plots the feature values on the x-axis and Shapley values on the y-axis, as well as

Reviewer: ```suggestion
Lastly, the `plot_type = "scatter"` plots the feature values on the x-axis and Shapley values on the y-axis, as well as
```

> (optionally) a background scatter_hist showing the distribution of the feature data. :

Reviewer: ```suggestion
(optionally) a background scatter_hist showing the distribution of the feature data:
```

plot(explanation_cat, plot_phi0 = FALSE, index_x_explain = c(1, 6))
```

We can specify parameters used to build the conditional inference trees

> using the following manner. Default values are based on

Reviewer: ```suggestion
in the following manner. Default values are based on
```
